llama_hub/wordlift/base.py

```diff
@@ -77,16 +81,20 @@ def fetch_data(self) -> dict:
         """
         try:
             query = self.alter_query()
-            response = requests.post(self.endpoint, json={
-                "query": query}, headers=self.headers)
+            response = await asyncio.to_thread(
+                requests.post,
+                self.endpoint,
+                json={"query": query},
+                headers=self.headers,
+            )
```

Comment:
What's the reason to run the request on another thread? or add async operations?